libs/aws/langchain_aws/chat_models/bedrock.py

```diff
@@ -439,7 +439,7 @@ def _stream(
         if "claude-3" in self._get_model():
             if _tools_in_params({**kwargs}):
                 result = self._generate(
-                    messages, stop=stop, run_manager=run_manager, **kwargs
+                    messages, stop=stop, run_manager=run_manager, stream=False, **kwargs
```

Comment:
Rather than adding a new parameter, can try adding to the kwargs. 
```python
kw = {**kwargs, "stream": False}
result = self._generate(
    messages, stop=stop, run_manager=run_manager, **kw
)
```